src/dos/drive_overlay.cpp

```diff
@@ -254,7 +257,7 @@ FILE* Overlay_Drive::create_file_in_overlay(const char* dos_filename, const char
 		f = fopen_wrap(newname,mode);
 	}
 
-	return f;
+	return std::make_pair(f, newname);
```

Comment:
You should be able to just write `return {f, newname};`.

Reply:
OK, changed.